blocks/graph.py

```diff
@@ -550,6 +550,17 @@ def apply_dropout(computation_graph, variables, drop_prob, rng=None,
         Random number generator.
     seed : int
         Random seed to be used if `rng` was not specified.
+    custom_divisor : float or None, optional
+        Divide dropped variables by a given scalar value. If `None`,
+        (default) dropped variables will be divided by `(1 - drop_prob)`
+        which is equivalent to scaling by `(1 - drop_prob)` at test
+        time as recommended in [DROPOUT]_.
+
+    Returns
+    -------
+    dropped_computation_graph : instance of :class:`ComputationGraph`
+        A new computation graph with dropout applied to the specified
+        variables.
```

Comment:
Since you are modifing dropout, can you mention, that one needs to extract new costs from a modified graph. Seems, sometimes people do not understand it.